### ISSUE TITLE
课文里是1998年

### ISSUE THREAD

**Issue by fhxshrs** [rdrg109/subtitles-npcr#79]
https://github.com/rdrg109/subtitles-npcr/blob/4905cef6c4c5326e03470cf75f4f055425e8b8d2/sentences/44-2.yaml#L2

我现在知道了有视频和文本两种版本，只是告诉你一声。

**Comment by rdrg109** [comment]
Done! 5f8f7fb

**Comment by rdrg109** [comment]
I just realised that the text and the video mention two different years. I think it's better to keep the two versions. One for the video and the other one for the text. In 291666a, I've added the version from the video.

For the record, here's an screenshot of the text.

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/ab7ba472-1754-4e73-aaaa-a4cfe44b44ad)

Here's the extract of the video that mentions "1992"

[output.webm](https://github.com/rdrg109/subtitles-npcr/assets/74389646/defb1c7e-bfe9-4cc3-9399-88aacad823e8)
